### ISSUE TITLE
Space between multiple nodes targeting single node

### ISSUE THREAD

**Issue by karolis-666** [kieler/elkjs#255]
Graph 1
<img width="581" alt="info1" src="https://github.com/kieler/elkjs/assets/79225490/6d644c46-dcaa-424d-abc5-456e17e4e901">

Graph 2
<img width="577" alt="info2" src="https://github.com/kieler/elkjs/assets/79225490/6eb799f9-7f65-4d1b-a878-9e58f5a3b339">

Elk options:
```
  'elk.algorithm': 'layered',
  'elk.direction': 'DOWN',
  'elk.spacing.nodeNode': '30',
  'elk.layered.spacing.nodeNodeBetweenLayers': '30',
  'elk.layered.spacing': '1',
  'elk.layered.mergeEdges': 'true',
  'elk.spacing': '20',
  'elk.spacing.individual': '20',
  'elk.edgeRouting': 'POLYLINE',
```

Seems that if multiple nodes target single node a big gap appears between them (Graph 1). Other example shows what gaps are expected between nodes (Graph 2). How can I reduce Graph 1 spacing like in Graph 2?

**Comment by soerendomroes** [comment]
We list our different spacing values [here](https://eclipse.dev/elk/documentation/tooldevelopers/graphdatastructure/spacingdocumentation.html) and all available option for the layered algorithm [here](https://eclipse.dev/elk/reference/algorithms/org-eclipse-elk-layered.html).
Additionally, I find it helpful to use the [elklive](https://rtsys.informatik.uni-kiel.de/elklive/elkgraph.html) (currently it seems to be down) tool to test different values since it has content assist.

TLDR: Try to use the `[edgeNodeBetweenLayers](https://eclipse.dev/elk/reference/options/org-eclipse-elk-layered-spacing-edgeNodeBetweenLayers.html)` spacing additionally to the `[nodeNodeBetweenLayers](https://eclipse.dev/elk/reference/options/org-eclipse-elk-layered-spacing-nodeNodeBetweenLayers.html)` spacing.

```
spacing.edgeNodeBetweenLayers: 1
spacing.nodeNodeBetweenLayers: 1

node n1

node n2

node n3

edge n1 -> n2

edge n1 -> n3
```

If you have further questions please feel free to ask and maybe this small SVG below does help. The spacings are abbreviated such that nodeNodeBetweenLayers is abbrevated to NNBL, etc.

![image](https://github.com/kieler/elkjs/assets/6419799/d5736fcd-6f8a-4310-acc1-3c8958e5ed65)

**Comment by karolis-666** [comment]
Thanks for reply, it's really helpful, but somehow changing these options value to lower number had opposite results towards bigger diagram. I've created sandbox with an example, maybe you can take a quick look, maybe you notice what I'm missing.

Codesandbox: [link](https://codesandbox.io/p/sandbox/diagrams-jqcx47?file=%2Fsrc%2FDiagram%2FDefaultNode.js%3A25%2C21&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clqeypbre0006356kc78x7xij%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clqeypbrd0002356kbls8eakb%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clqeypbre0003356k3bu0zbo3%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clqeypbre0005356khcldl9qf%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B40.2995017171755%252C59.7004982828245%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clqeypbrd0002356kbls8eakb%2522%253A%257B%2522id%2522%253A%2522clqeypbrd0002356kbls8eakb%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqeypbrd0001356ka6hqwl5z%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522clqeziai60002356kkwug9tvm%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A17%252C%2522startColumn%2522%253A3%252C%2522endLineNumber%2522%253A17%252C%2522endColumn%2522%253A3%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252FDiagram%252Findex.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522clqf0sfu70002356k5el6rdgy%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A25%252C%2522startColumn%2522%253A21%252C%2522endLineNumber%2522%253A25%252C%2522endColumn%2522%253A21%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252FDiagram%252FDefaultNode.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clqf0sfu70002356k5el6rdgy%2522%257D%252C%2522clqeypbre0005356khcldl9qf%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqeypbre0004356kpszognxe%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clqeypbre0005356khcldl9qf%2522%252C%2522activeTabId%2522%253A%2522clqeypbre0004356kpszognxe%2522%257D%252C%2522clqeypbre0003356k3bu0zbo3%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clqeypbre0003356k3bu0zbo3%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D).

**Comment by soerendomroes** [comment]
@karolis-666 I have missed your last question, I guess since you closed the ticket, the issue is resolved?

**Comment by karolis-666** [comment]
Yes, most of issues managed to solve by changing `edgeRouting` to `SPLINES`

**Comment by soerendomroes** [comment]
This sounds odd. I did a little investigation and the issue seems to be that the `edgeEdgeBetweenLayers` seems to have an effect on the drawing if `edgeRouting` is set to `POLYLINE`, as seen [here](https://rtsys.informatik.uni-kiel.de/elklive/json.html?compressedContent=N4KABGCWAmBcYCIBOB7FAXBAacYA2AhgJ4oCu6A8gA7qQoB2AzvKBG4gKZ4DWAdAXgDmKJJHQALALYJ4CQkQ5IO0bLggIufaJCUBjWgxmIAIhQDqAOVXsNPXvMXLejKgV2R6g3vRTQOF3w4AIQ50AHcODnoAGWJFRiMEAGYABmwwAHoMsAD0MCU8DgA3Ano8xg9dDjBlQWqfPzAAKjAAJihGMAAjSEE6pDVOOxc3Dy8G-0DE1Os2Wz4HJWhnV3dPXlrJvxDwyJi4pATZGaxM7IBlVbHu0IiosAAzHUY8osVaXQEa6DqwRg5BJIonlStAwBNGKdxFEqmAJNVwigwK4kLQDEwwOICG8wAQ-ldPGAUA8wAA2FKDeb2A5OEZrLybACiP2Ctz2sQUh0SAEYUnz0lkwOdxGQ8GCfHkutUCsVSpLyB1HiIwAAFCjRACa0QAkhZGYMAOSaDYsgBKZFongN8ANas1Or1BpwEFwAF9nWBdOJIGKlPR4ABtQbAKBwRD0bnpMIwCTwVJQji9cToOMpMDu4Oh2T0VpRmPiVMJpMpsCpdMeiAhmDZ1qR07R6Cx0spIuCZOp8uZ6vh1q5+v5wuYxNtktljPsKthhA5pJ5xsF5ut9vNzsTrM9gAsc6b8aHxY747Yk5rAFZtwvd9D9yvD5X19PZ-354OryOD7gALoezbMMBBtfdhodZ-GQSBVL+AbhpGYBfnCBBIHU6AQeGs4wauR73hwtbpIwoHgYGPbQbB6DwYhyEPggaG3mAx6cL2OF4Rw5E5rmMGnCRCGhMxqEfuhd6AVhj4gaQYFMQR06tDx7GkVx4n0DxfE0ZhrRbqcuEifhf6bpRxEyUhckKdRtEaK0Z5qYxzGmTp0mcfpWkUVRFZKQJfbCaJlnWXBtncTp6afiArpAA).
